crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -255,6 +437,63 @@ def proposals
   api :GET, "/crowbar/:barclamp/1.0/proposals/template",
     "Returns the content of a proposal template"
   param :barclamp, String, desc: "Name of the barclamp", required: true
+  example '
+  {
+    "id": "template-dns",
```

Comment:
in an ideal world I agree, that'd be super useful. but this would be an enormous amount of documentation, just imagine a proposal alone, and then repository metadata etc. we don't have the time to do that now